zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java

```diff
@@ -50,7 +52,13 @@ public ZuliaDConfig(String config) throws IOException {
 
 		LOG.info("Loading config <" + config + ">");
 
-		zuliaConfig = GSON.fromJson(new FileReader(config), ZuliaConfig.class);
+		if (config.endsWith("json") || config.endsWith("properties")) {
```

Comment:
should we do config.toLower here?

Reply:
I mean... we could but I would say capital filenames are wrong.